### ISSUE TITLE
Unable to detect surql-gen.json in current folder

### ISSUE THREAD

**Issue by limcheekin** [sebastianwessel/surrealdb-client-generator#1]
Hi there,

Thanks for your kindness of created this productive tool and share it here.

Here is my [surql-gen.json](https://github.com/sebastianwessel/surrealdb-client-generator/files/12903507/surql-gen.json) file, I not sure how the `surql-gen.json` file looks like, if you could provide a sample, it will be helpful.

I tried the following commands:
```
npx surql-gen
npx surql-gen -c surql-gen.json 
npx surql-gen -c ./surql-gen.json 
```

I always see the following message in the console:
```
No config file provided - looking for surql-gen.json in current folder
```

The `surql-gen.json` was indeed located in the current directory, I not sure why it doesn't pick up by surql-gen.

My system information:
```
Ubuntu 22.04.3 LTS
node v20.5.1
npm 9.8.0
```

Please advise. Thank you.

**Comment by sebastianwessel** [comment]
Thanks for using this piece of software.
There was an issue with reading and using the CLI parameter, which should be fixed in recent version 1.1.0.

Maybe you can try out with the new version @limcheekin

**Comment by limcheekin** [comment]
Thanks for quick turnaround!

I did a simple testing using `-s` option from the CLI like the following:
```
npx surql-gen -f ../rag/schema.surql 
```

It works! :)
